plugins/processors/geo/README.md

```diff
@@ -0,0 +1,32 @@
+# Geo Processor Plugin
+
+Use the `geo` processor to add tag with S2 cell ID token of specified [`cell level`][cell levels].
+
+The tag is used in `experimental/geo` Flux package functions.
+
+### Configuration
+
+```toml
+[[processors.geo]]
+  ## The name of the lat and lon fields
+  lat_field = "lat"
+  lon_field = "lon"
```

Comment:
Add more documentation about the precise required format of these fields.

Reply:
these are floats...and the idea is to allow the user to identify the names of the inbound fields which represent the latitude and longitude field names.

Reply:
Something like this then:
```toml
## Name of the float fields containing latitude and longitude in decimal degrees.
latitude_field = "lat"
longitude_field = "lon"
```